pulp_2_tests/tests/rpm/utils.py

```diff
@@ -138,6 +138,17 @@ def check_issue_4405(cfg):
             not selectors.bug_is_fixed(4405, cfg.pulp_version))
 
 
+def check_issue_4529(cfg):
+    """Return true if `Pulp #4529`_ affects the targeted Pulp system.
+
+    :param cfg: The Pulp system under test.
+
+    .. _Pulp #4405: https://pulp.plan.io/issues/4529
```

Comment:
```suggestion
    .. _Pulp #4529: https://pulp.plan.io/issues/4529
```

Reply:
Missed this on the `git add`. Already in place.

Sorry about that.